experimental/generation/generator/packages/library/src/processSchemas.ts

```diff
@@ -184,23 +183,26 @@ const ContraintKeywords = ['minimum', 'maximum', 'exclusiveMinimum', 'exclusiveM
  * This makes it easier to use a child schema in items.
  * @param schema Schema fragment to promote.
  */
-function promoteItems(schema: any): void {
+function promoteItemsAndRemoveGenerator(schema: any): void {
```

Comment:
I'd suggest using `unknown` instead of `any`, especially since you're already doing runtime type checks in the function body.

Reply:
This whole code base has made use of any for working with schema.  Could be a good idea to switch to unknown at least for new code.